qemu/tests/readonly_disk.py

```diff
@@ -32,16 +31,33 @@ def run(test, params, env):
         "TEST STEPS 2: Format the disk and copy file to it", logging.info)
     os_type = params["os_type"]
     copy_cmd = params.get("copy_cmd", "copy %s %s")
-    disk_idx = params.get("disk_index", 1)
-    fs_type = params.get("fstype", "ntfs")
-    drive_letter = params.get("drive_letter", "I:")
-    disk_size = params.get("partition_size_data", "200M")
+    fstype = params.get("fstype", "ntfs")
+    data_image_size = params.get("image_size_data", "1G")
+    data_image_num = int(params.get("data_image_num",
+                                    len(params.objects("images")) - 1))
+    err_msg = "Set disks num: %d" % data_image_num
+    err_msg += ", Get disks num in guest: %d"
+    error_context.context("Get windows disk index that to "
+                          "be formatted", logging.info)
+    disks = utils_disk.get_windows_disks_index(session, data_image_size)
+    if len(disks) < data_image_num:
```

Comment:
for this case ,the disks list should only has 1 cell, why not condition if len(disks) == 0? can reduce param data_image_num

Reply:
this condition used universally